rust/src/index/vector/ivf.rs

```diff
@@ -584,6 +588,7 @@ pub async fn build_ivf_pq_index(
         Ivf::new(centroids.clone())
     } else {
         // Pre-transforms
+        #[cfg(feature = "opq")]
```

Comment:
Let’s at a block where when opq feature is not installed, generate an error if uses passes use_opq=true

Reply:
It looks like this was added to Python, but not yet Rust? Could you add that error to the rust function here as well?